.github/workflows/e2e-android.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           export PATH=$ANDROID_SDK_ROOT/platform-tools:$PATH
           export PATH=$ANDROID_SDK_ROOT/emulator:$PATH
           cd packages/mobile
-          yarn run test:e2e:android -w 1
+          yarn run test:e2e:android -w 3
```

Comment:
just checking if this can be 5, like it is for iOS?

Reply:
I've bumped it up to 4 in https://github.com/valora-inc/wallet/pull/1829/commits/9169c06c5cc38f81e7b971ac544213e937ddde28. When I bumped it up to five we actually had slower runs possibly due to a bottlenecking of resources. [Android Emulators vs iOS Simulators - Stack Overflow](https://stackoverflow.com/a/4544605/7967484).

Reply:
Move it back down to 3 in https://github.com/valora-inc/wallet/pull/1829/commits/23801a087355c642fb2d3a28cbc39fa95086b5c7. This gives us 4 gb per emulator and 4 gb of overhead for our CI machines.